index.js

```diff
@@ -201,18 +203,21 @@ var animationQueue = new AnimationQueue();
 // Give each notification a unique id
 var latestID = 0;
 
-function notify(title, text, url, image, onClickFunc, onShowFunc, onCloseFunc) {
+function notify(title, text, url, image, sound, onClickFunc, onShowFunc, onCloseFunc) {
```

Comment:
true, however this function should have dynamic arg's anyways and currently it's not designed that way but it should, could implement that for the next pr though.